synfig-studio/src/gui/resourcehelper.cpp

```diff
@@ -0,0 +1,104 @@
+/* === S Y N F I G ========================================================= */
+/*!	\file resourcehelper.h
+**	\brief Helper to retrieve the app resource paths, such as icons and plugins
+**
+**	$Id$
+**
+**	\legal
+**	Copyright (c) 2002-2005 Robert B. Quattlebaum Jr., Adrian Bentley
```

Comment:
@rodolforg Please add your copyright here. ^__^

Reply:
Oh, I really don't care. Git commit logs are enough. lol
Maybe we should do like others projects.
In Godot Engine, for example:
```
/* Copyright (c) 2007-2019 Juan Linietsky, Ariel Manzur.                 */
/* Copyright (c) 2014-2019 Godot Engine contributors (cf. AUTHORS.md)    */
```

Reply:
Added it anyway

Reply:
Anyway I added it and rebase it (in order to not create a commit just for copyright)

Reply:
> Oh, I really don't care. Git commit logs are enough. lol
> Maybe we should do like others projects.
> In Godot Engine, for example:
> ...

This is actually a great idea! I like it. ^__^